app/assets/scripts/views/projects-hub/project-card.js

```diff
@@ -39,18 +40,32 @@ export default function ProjectCard({
         <CardDetails
           id="project"
           list={[
-            { label: 'Locations', value: formatThousands(totalLocations) },
+            { label: 'Collection dates', value: `${started} - ${ended}` },
             {
               label: 'Measurements',
               value: formatThousands(totalMeasurements),
             },
-            { label: 'Collection dates', value: `${started} - ${ended}` },
             {
               label: 'Parameters',
               value: parametersList
                 .map(p => p.parameter.toUpperCase())
                 .join(', '),
             },
+            { label: 'Locations', value: formatThousands(totalLocations) },
+            {
+              label: 'Source',
+              value:
+                sources &&
```

Comment:
Should we display a fallback in case there are no sources listed?

Reply:
@AliceR is that possible, i thought a project is based on a source? happy to add, just wondering if it is redundant

Reply:
I am not sure if it should be possible, I just noticed that currently the source is blank for all the project cards.

Reply:
@AliceR honestly im not sure if i fixed it or something changed on the api, but theyre there now!